aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -158,6 +159,7 @@ SparseTensor new_with_size_sparse(IntList size, const TensorOptions& options) {
 // are guaranteed to be within bounds.
 // NB: Got rid of the sizes == NULL case
 SparseTensor new_with_tensor_and_size_unsafe_sparse(const LongTensor& indices, const Tensor& values_, ArrayRef<int64_t> sizes) {
+  AT_ASSERT(indices.is_contiguous() && values_.is_contiguous());
```

Comment:
No you shouldn't assert these! These are provided by users. If you want to error out, use AT_CHECK.